pyproject.toml

```
@@ -58,16 +58,28 @@ moto = "^4.1.3"
 boto3 = "^1.26.78"
 SQLAlchemy = "<=3.0.0"
 
+# Example to create requirements.txt for lambda layers:
+# poetry export -f requirements.txt --with lambda-database -o
+# lambda_layer/database/requirements.txt --without-urls
+[tool.poetry.group.lambda-sdc]
```

can you rename this to something like `lambda-imap-processing`? because my mind thinks that `lambda-sdc` should contain everything that SDC needs which are what's in `lambda-database`.

while we are it, should we rename all `lambda` to `layer` instead? Something like this. This way it kind of matches how we are calling it in the code or how it's used.
```suggestion
[tool.poetry.group.imap-processing-layer]
```

---

Ok. I renamed everything from lambda to layer